Mergin/plugin.py

```diff
@@ -369,6 +358,96 @@ def download(self):
         )
         if btn_reply == QMessageBox.Yes:
             self.open_project()
+        self.parent().reload()
+
+    def clone_remote_project(self):
+        user_info = self.mc.user_info()
+        dlg = CloneProjectDialog(username=user_info["username"], user_organisations=user_info.get("organisations", []))
+        if not dlg.exec_():
+            return  # cancelled
+        try:
+            self.mc.clone_project(self.project_name, dlg.project_name, dlg.project_namespace)
+            msg = "Mergin project cloned successfully."
+            QMessageBox.information(None, "Clone project", msg, QMessageBox.Close)
+            self.parent().reload()
+            # we also need to reload My projects group as the cloned project could appear there
+            group_items = self.project_manager.get_mergin_browser_groups()
+            if "My projects" in group_items:
+                group_items["My projects"].reload()
+        except (URLError, ClientError) as e:
```

Comment:
I would prefer to keep the try/except blocks as short as possible, so it is clear where the error is coming from - e.g. here it should be enough to just wrap the clone_project() call?